jstz_api/src/console.rs

```diff
@@ -22,10 +22,41 @@ use boa_engine::{
     value::Numeric,
     Context, JsArgs, JsNativeError, JsResult, JsValue, NativeFunction,
 };
-use boa_gc::{Finalize, GcRefMut, Trace};
+use boa_gc::{empty_trace, Finalize, GcRefMut, Trace};
 use jstz_core::{host::HostRuntime, runtime, value::IntoJs};
+use jstz_crypto::{hash::Blake2b, public_key_hash::PublicKeyHash};
+use serde::{Deserialize, Serialize};
+use serde_json;
 use tezos_smart_rollup::prelude::debug_msg;
 
+#[derive(Serialize, Deserialize)]
+struct JSONLog {
+    function_address: String,
+    request_id: String,
+    level: String,
+    text: String,
+}
```

Comment:
Rust naming: `JSONLog` to `JsonLog`. 
Additionally I'd prefer to avoid having `JSON` in the name, since in future the encoding is likely to change. Perhaps `LogRecord`?